polynote-kernel/src/main/scala/polynote/kernel/ScalaCompiler.scala

```diff
@@ -68,6 +70,13 @@ class ScalaCompiler private (
       }
   }
 
+  private object saveOriginalPos extends Traverser {
```

Comment:
FWIW, I didn't end up needing this, but I think it's going to save lives one day to save the original position in an attachment (what this does). If I'm wrong, it's easy to remove this.